cmake/FindThreads.cmake

```diff
@@ -0,0 +1,250 @@
+# - This module determines the thread library of the system.
```

Comment:
You are right, tested it only on my local linux machine. I will create my own PR and try it out on appveyor without that file.